api/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/api/innsending/InnsendingProducer.kt

```diff
@@ -20,10 +20,15 @@ class InnsendingProducer(
         val packet: JsonMessage = JsonMessage.newMessage(
             mapOf(
                 "@event_name" to "inntektsmelding_inn",
-                "@behov" to listOf(BehovType.JOURNALFOER.name),
+                "@behov" to listOf(
+                    BehovType.VIRKSOMHET.name
+                ),
+                "@extra" to BehovType.JOURNALFOER.name,
```

Comment:
Jeg ser ikke helt hvordan framgangsmåten skalerer for løsere i serie. Her fungerer det vel bare med maks to?

Reply:
Antall behov har ingen begrensninger. Foreløpig er extra feltet kun 1 stk.

Reply:
Tenker at vi kan utvide etterhvert. Om behovet dukker opp.

Reply:
La inn mulighet for ubegrenset antall løsere i serie.